### DISCUSSION TITLE
asus A15 CPU power limit suggestion

### DISCUSSION THREAD

**Discussion by santhoshsriram24** [seerge/g-helper#3021] [Q&A] [answered]
hi recently i got to know about g helper from my friend and i give it a try it was so good , i have doubt i dont know that the cpu power limit and undervolting is working for me or not , and a small favour can u suggest a setting for a good battery life browsing and coding ?
![Screenshot 2024-08-26 183854](https://github.com/user-attachments/assets/1798abf3-733f-4ee1-840a-0b39abdc3658)
![Screenshot 2024-08-26 183912](https://github.com/user-attachments/assets/35f9358c-72f9-41f4-9cd6-aced7716f44f)

**Comment by seerge** [comment, marked as answer]
@santhoshsriram24 hello. On you screenshot CPU consumes 3.5W-6.7W which is below a limit you have set, so everything is correct.

For undervolting - don't expect a miracle. You can just run Cinebench23 with UV set to 0 and with UV set to -10 (and other thins exactly same) and compare scores. Something like 1-2% more in a score is a good result
